lib/metabase/endpoint/pulse.rb

```diff
@@ -11,6 +11,15 @@ module Pulse
       def pulses(**params)
         get('/api/pulse', **params)
       end
+
+      # Delete a subscription.
+      #
+      # @param params [Hash] Query string
+      # @return Parsed response JSON
+      # @see https://github.com/metabase/metabase/blob/master/docs/api-documentation.md#get-apipulse
```

Comment:
Delete subscription document url is https://github.com/metabase/metabase/blob/master/docs/api/pulse.md#delete-apipulseidsubscription